src/wp-includes/class-wp-text-diff-renderer-table.php

```diff
@@ -16,6 +16,10 @@
 #[AllowDynamicProperties]
 class WP_Text_Diff_Renderer_Table extends Text_Diff_Renderer {
 
+	protected $_title;
```

Comment:
Could you please add document for each variable?

Reply:
Sorry, but I'm not familiar enough with the process. Please anyone, feel free to go ahead.